src/inc/DefaultSettings.h

```diff
@@ -35,6 +35,7 @@ constexpr int DEFAULT_FONT_SIZE = 12;
 
 constexpr int DEFAULT_ROWS = 30;
 constexpr int DEFAULT_COLS = 120;
+constexpr int DEFAULT_ROWSTOSCROLL = NULL;

```

Comment:
nit: I think we're trying to move away from `NULL` here - I'd prefer either `nullptr` or even `0` (probably the later, since this isn't really a case where `nullptr` applies

Reply:
Yeah Null expands to 0 anyway, so I'll change all the Null values to 0.